config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: example.com/thanos-operator
-  newTag: v0.0.1
+  newName: quay.io/thanos/thanos-operator
+  newTag: main-2024-09-27-8c9f051
```

Comment:
Want to revert this one? As otherwise tag would be outdates. The example.com image is generated in tests

Reply:
can we build a latest tag off main?

Reply:
We do build it, but this file is used in e2e tests.